### ISSUE TITLE
Enemies at top of map may be hidden by combat bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#174]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1.Find an enemy on the top of a map (i.e. one of the bandits in the vacor quest)
2.Attack the enemy
3.

What is the expected output? What do you see instead?
The combat bar obscures the enemy. You can still fight it, but it is 
"off-screen"

What version of the product are you using? On what device?
v0.6.8

Please provide any additional information below.
This doesn't happen very often, so it's not a huge issue.

```

Original issue reported on code.google.com by `jch...@email.wm.edu` on 5 Apr 2011 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
Addendum: I am on a Droid Incredible
```

Original comment by `jch...@email.wm.edu` on 5 Apr 2011 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. This is similar to issue 106.

We should go through the maps and spot where spawn areas are placed too close 
to the top edge.
```

Original comment by `oskar.wi...@gmail.com` on 7 Apr 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Oh, I missed that one. Thanks for the link. Also, thanks for a great game, btw!
```

Original comment by `jch...@email.wm.edu` on 8 Apr 2011 at 4:35

**Comment by GoogleCodeExporter** [comment]
```
This is now partly solved by the excellent update of the combat interface done 
by Juve as part of r71. The combat interface background is now 
semi-transparent, which allows you to view the combat een if it is behind the 
combat bar(s).
```

Original comment by `oskar.wi...@gmail.com` on 14 Apr 2011 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
We should close this issue as there is a duplicate:
http://code.google.com/p/andors-trail/issues/detail?id=130
```

Original comment by `SamuelPl...@gmail.com` on 9 Jul 2011 at 6:40

**Comment by GoogleCodeExporter** [comment]
```
Closing, fixed in v0.6.9.
```

Original comment by `oskar.wi...@gmail.com` on 13 Jul 2011 at 8:33
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.9**
